js/run.js

```diff
@@ -127,6 +127,8 @@ $(document).ready(function()
     {
         var root = $(this);
         var el = root.children("pre");
+        // hack from the front page to avoid redraw flashes due to the buttons showing up
+        el[0].style.marginBottom = "0px";
```

Comment:
Instead of removing the margin, can this be left in? As mentioned in the previous PR, my suggestion was to simply create a container (whose size is not going to change) for the buttons to be placed in. I think that would avoid a relayout.